PROJECT

```diff
@@ -288,4 +288,12 @@ resources:
   kind: ProviderInterface
   path: github.com/k0rdent/kcm/api/v1beta1
   version: v1beta1
+- api:
+    crdVersion: v1
+    namespaced: false
```

Comment:
nit: i was under the impression if a resource is cluster wide, then the field is actually omit by the `kubebuilder`